template.xml

```diff
@@ -120,8 +120,8 @@
          <t hangText="Required parameters:">n/a</t>
          <t hangText="Optional parameters:">n/a</t>
          <t hangText="Encoding considerations:">binary</t>
-         <t hangText="Security considerations:">See section 9 above</t>
-         <t hangText="Interoperability considerations:">See section 10 above</t>
+         <t hangText="Security considerations:">See section 10 above</t>
+         <t hangText="Interoperability considerations:">See section 11 above</t>
```

Comment:
and `<xref target="interoperability-considerations" pageno="false" format="default"/>` here.